content/guides/references/migration-guide.md

```diff
@@ -2,6 +2,151 @@
 title: Migration Guide
 ---
 
+## Migrating to Cypress 10.0
+
+This guide details the changes and how to change your code to migrate to Cypress
+10.0. [See the full changelog for 8.0](/guides/references/changelog#10-0-0).
+
+## Attaching Files
+
+Attaching files to input elements or dropping them over the page is available
+in Cypress 10.0. Read the [`cy.attachFile()` API docs](/api/commands/attachFile)
```

Comment:
Camelcase will 404 here

api/commands/attachfile
```suggestion
in Cypress 10.0. Read the [`cy.attachFile()` API docs](/api/commands/attachFile)
```
```suggestion
in Cypress 10.0. Read the [`cy.attachFile()` API docs](/api/commands/attachFile)
```

Reply:
Fixing.

Note that urls are actually case-insensitive - https://DOCS.cyPrEss.iO/api/cOmMaNdS/wItHiN works just fine. :D

Reply:
Ah ok. Likely because I was looking at this before the addition to the sidebar.json and assumed that was the issue. I'm going to write all my URLs in this fashion moving forward 😆